gitversion/gen_cpp_header/gen_version_header.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python2
+
+from jinja2 import Environment, FileSystemLoader
+from collections import OrderedDict 
+import json
+
+file_loader = FileSystemLoader('template')
+env = Environment(trim_blocks=True, lstrip_blocks=True, loader=file_loader) 
+
+template = env.get_template('template_version.h')
```

Comment:
make this as command line argument for python script.

Reply:
ok, fixed!